### PR TITLE
ON-4070 | Page meta data not editable

### PR DESCRIPTION
**Pull request by rob-mcclain-swarm**
If another component is selected when a head/invisible component settings form is opened, the form will use that other component's schema for some field data.  This will cause the head component fields to be disabled if fields by the same name do not exist in the other component.  The fix here is to dispatch an unselect action when opening the form.

### CODE REVIEW

File: lib/drawers/head-components.vue [hunk 1]
```diff
@@ -72,6 +72,9 @@
       openSettings(id) {
         const path = 'settings';
 
+        // If a different element is selected, the overlay form will use the
```
Comment (csdswarm) [inline lib/drawers/head-components.vue]: I'd shorten this to something like "unselect to prevent overlay from using wrong schema"

**Comment by csdswarm** [comment]
I'm good. Approved.

**Review by laddhoffman-swarm — APPROVED** [review]
